server/routes/index.js

```diff
@@ -0,0 +1,15 @@
+const keystone = require('keystone');
+const importRoutes = keystone.importer(__dirname);
+
+const routes = {
+  api: importRoutes('./api')
+};
+
+exports = module.exports = function(app) {
+  app.get('/api/recipe/list', keystone.middleware.api, routes.api.recipes.list);
+
+  // app.get('/api/route', (err, get) => {
```

Comment:
What about this comments?

Reply:
Those commets are just guidelines, i'll erase them all